### DISCUSSION TITLE
How to keep a Group Rule after updating a Config Update?

### DISCUSSION THREAD

**Discussion by SuperCoolDude** [vernesong/OpenClash#3014] [Q&A]
Hello, I created a custom Policy Group, but whenever I refresh my my subscriptions with a Config Update, I lose the group.

Is there a way to keep the custom Policy Group between Config Updates? Or do I need to manually add it back in each time by editing the .yaml file?

**Comment by donf6938** [comment]
Try use proxy-providers
